{"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/com.365id.sdk.idverification\/documentation\/IdVerification365id\/IdVerificationError\/clientError"},"sections":[],"schemaVersion":{"major":0,"minor":3,"patch":0},"primaryContentSections":[{"declarations":[{"languages":["swift"],"tokens":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"text":"clientError","kind":"identifier"}],"platforms":["iOS"]}],"kind":"declarations"}],"hierarchy":{"paths":[["doc:\/\/com.365id.sdk.idverification\/documentation\/IdVerification365id","doc:\/\/com.365id.sdk.idverification\/documentation\/IdVerification365id\/IdVerificationError"]]},"abstract":[{"text":"Happens when there is an error caused by something on the device itself","type":"text"}],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/idverification365id\/idverificationerror\/clienterror"]}],"metadata":{"roleHeading":"Case","externalID":"c:@M@IdVerification365id@E@IdVerificationError@IdVerificationErrorClientError","title":"IdVerificationError.clientError","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"text":"clientError","kind":"identifier"}],"symbolKind":"case","modules":[{"name":"IdVerification365id"}],"role":"symbol"},"kind":"symbol","references":{"doc://com.365id.sdk.idverification/documentation/IdVerification365id/IdVerificationError/clientError":{"url":"\/documentation\/idverification365id\/idverificationerror\/clienterror","type":"topic","abstract":[{"text":"Happens when there is an error caused by something on the device itself","type":"text"}],"identifier":"doc:\/\/com.365id.sdk.idverification\/documentation\/IdVerification365id\/IdVerificationError\/clientError","kind":"symbol","title":"IdVerificationError.clientError","role":"symbol","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"text":"clientError","kind":"identifier"}]},"doc://com.365id.sdk.idverification/documentation/IdVerification365id":{"role":"collection","abstract":[],"identifier":"doc:\/\/com.365id.sdk.idverification\/documentation\/IdVerification365id","title":"IdVerification365id","url":"\/documentation\/idverification365id","kind":"symbol","type":"topic"},"doc://com.365id.sdk.idverification/documentation/IdVerification365id/IdVerificationError":{"kind":"symbol","identifier":"doc:\/\/com.365id.sdk.idverification\/documentation\/IdVerification365id\/IdVerificationError","type":"topic","url":"\/documentation\/idverification365id\/idverificationerror","role":"symbol","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"IdVerificationError"}],"navigatorTitle":[{"text":"IdVerificationError","kind":"identifier"}],"title":"IdVerificationError","abstract":[{"text":"Id Verification Errors used in the ","type":"text"},{"type":"codeVoice","code":"onError(error:)"},{"type":"text","text":" method in the "},{"type":"codeVoice","code":"IdVerificationEventDelegate"}]}}}